projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-vc.ts

```diff
@@ -291,6 +292,16 @@ export function clickOnViewInConfiguration(attribute: string): void {
   clickOnConflictSolverLink(attribute, 'iew in Configuration Link');
 }
 
+/**
+ * Navigates to a group that contains an attribute which is involved in a conflict and waits for request to finish
+ *
+ * @param attribute - Attribute name
+ */
+export function clickOnViewInConfigurationAndWait(attribute: string): void {
```

Comment:
@Uli-Tiger, could you add via which link will be clicked to navigate?
Otherwise, both methods `clickOnViewInConfigurationAndWait` and  `clickOnConflictDetectedAndWait` have the same description.